Source/Rows/Controllers/SelectorViewController.swift

```diff
@@ -210,8 +210,8 @@ open class _SelectorViewController<Row: SelectableRowType, OptionsRow: OptionsPr
                 }
             }
         }
-        for option in options {
-            section <<< Row.init(String(describing: option)) { lrow in
+        for (index, option) in options.enumerated() {
+            section <<< Row.init(String(describing: option) + String(index)) { lrow in
```

Comment:
I think prefixing the tag with the index makes more sense. It is also still possible to have duplicate tags but that would be less common

Reply:
Hi @mats-claassen How do you think about just using index as the tag? That should be unique and easy to read. Do you have any scenario in mind where it helps to include option value in the tag?

Reply:
So I go ahead to use only option index as row tag value. Please look at the new diff. Let me know if you'd like to include option string in the tag. Thanks.

Reply:
I'm sorry. I suppose only reviewers (not code change author) can click on "resolve conversion" button, right? I have clicked it again to "unresolve" it.